examples/pip-venv-kernel-image/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.11.1
+
+RUN pip install ipykernel && \
+        python -m ipykernel install --sys-prefix
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
where are we creating venv? I am not so familiar with venv. Other than the naming, it appear packages are installed into system python, right?

Reply:
Yes, this is the case. You don’t nee to initialize a new venv but use the default. This is pretty much the same principle as using the tf23-image sample code, but you start from base Python image here.